### ISSUE TITLE
`Docformatter` conflict with `isort`

### ISSUE THREAD

**Issue by M1troll** [PyCQA/docformatter#203]
Hi!

# Description

I recently update docformatter `v1.6.0`  -> `v1.6.4` and actions on my project was failed(
After spending time searching, reason turned out to be following - `isort` and `docmatter` make opposite changes to the file:
* `docmatter` - remove empty line between import and constant comment
* `isort` - add this line)

Also was checked `v1.6.1`, `v1.6.2` and `v1.6.3` - they have same problem.

As I see it, [PEP8](https://peps.python.org/pep-0008/#imports) says:
>You should put a blank line between each group of imports. 

I think this applies to constants as well, so `docmatter` shouldn't remove that line.

# Reproduction
You can reproduce my problem with next data:

Python code:
```python
import os
from typing import Iterator

"""Don't remove this comment, it's cool."""
IMPORTANT_CONSTANT = "potato"

```
`.pre-commit-config.yaml`:
```yaml
repos:
  - repo: https://github.com/pycqa/docformatter
    rev: v1.6.4
    hooks:
      - id: docformatter
        args: [
          --wrap-summaries=120,
          --wrap-descriptions=0,
          --in-place,
          --blank
        ]

  - repo: https://github.com/pycqa/isort
    rev: 5.12.0
    hooks:
      - id: isort

```

After run ```pre-commit run --all-files``` you will see that file changed twice and actions was failed:
```diff
import os
from typing import Iterator
-                                                                           
"""Don't remove this comment, it's cool."""
IMPORTANT_CONSTANT = "potato"

```
```diff
import os
from typing import Iterator
+                                                                           
"""Don't remove this comment, it's cool."""
IMPORTANT_CONSTANT = "potato"

```

**Comment by weibullguy** [comment]
@M1troll should be fixed in v1.6.5.

**Comment by M1troll** [comment]
@weibullguy , Thank you! Now everything works without problems)
